src/test/regress/expected/multi_modifications.out

```diff
@@ -874,6 +874,10 @@ INSERT INTO raw_table VALUES (2, 400);
 INSERT INTO raw_table VALUES (2, 500);
 INSERT INTO summary_table VALUES (1);
 INSERT INTO summary_table VALUES (2);
+-- test noop deletes
+DELETE FROM summary_table WHERE false;
```

Comment:
could be nice to have one test for update,  in case we one  day have different ways of  planning  it